spec/features/user_visits_homepage_spec.rb

```diff
@@ -6,9 +6,12 @@
 
     within("table.redirections") do
       expect(page).to have_content("gabebw")
-      expect(page).to have_content("http://gabebw.com")
       expect(page).to have_content("edward")
-      expect(page).to have_content("http://edwardloveall.com")
+
+      %w(http://gabebw.com http://edwardloveall.com).each do |url|
+        expect(page).to have_content(url)
+        expect(page).to have_css("a[href='#{url}']")
```

Comment:
It's a CSS selector for an `a` element with an `href` attribute with the url as it's value